cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -348,6 +384,17 @@ private predicate stateBarrier(NodeEx node, FlowState state, Configuration confi
   exists(Node n |
     node.asNode() = n and
```

Comment:
Code Scanning is not wrong here, I think 😂. I think this is the change you want to make?
```suggestion
    node.asNode() = n |
```

Reply:
Indeed. Thanks.